lib/application.js

```diff
@@ -145,7 +147,8 @@ app.lazyrouter = function lazyrouter() {
   if (!this._router) {
     this._router = new Router({
       caseSensitive: this.enabled('case sensitive routing'),
-      strict: this.enabled('strict routing')
+      strict: this.enabled('strict routing'),
+      maxURLLength: this.get('maxURLLength'),
```

Comment:
Actually looking again, should we use the space pattern instead of camel case?

Reply:
in the end I opted not to because:

- express API options are camelCase
- express app settings are lower sentence case (with one exception)
- this change introduces both a new API option and a new app setting, but they are for the same thing (url length limit)
- using a different key for the option vs the setting seems like the worst option, so we should pick one or the other. this means either the settings will have an outlier in the form of camelCase, or the options will have an outlier in the form of lower sentence case
- the camelCase key is preferred over sentence case for a few reasons, but also it would be annoying to write and to read `var router = new express.Router({ 'maximum URL length': 4096 });`

Reply:
totally valid. Ok you convinced me. I never liked those settings as sentence case anyway lol.